frontend/src/app/home/home.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { NgxUiLoaderService } from 'ngx-ui-loader';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss']
+})
+export class HomeComponent implements OnInit {
+
+  constructor(private ngxService: NgxUiLoaderService) { }
+
+  ngOnInit() {
+    this.ngxService.start();
+    setTimeout(() => {
+      this.ngxService.stop();
+    }, 10000);
```

Comment:
To sztywne ustawienie jest kiepskim rozwiązaniem. Przy najprostrzym loaderze, najlepiej podpiąć się interceptorem po http requesty. 
Np. ta biblioteka daje nam to z pudełka:
`https://www.npmjs.com/package/ng-http-loader`